src/testjfxapp/Scoreboard.java

```diff
@@ -14,6 +20,8 @@
     Button btm1;
 
     MainMenu mainMenu;
+    Tetris tetris = new Tetris(10, 20, 30, mainMenu);
+    // Add same for tetsaw class
```

Comment:
Do I need to explain again why this is not what you should be doing?
Add a Tetris object to your constructor's inputs, and assign the variable to that. You should only ever make a new Tetris object when you are starting a new game.